src/raftstore/store/worker/apply.rs

```diff
@@ -919,13 +919,15 @@ impl ApplyDelegate {
         request: &AdminRequest,
     ) -> Result<(RaftCmdResponse, Option<ExecResult>)> {
         let cmd_type = request.get_cmd_type();
-        info!(
-            "{} execute admin command {:?} at [term: {}, index: {}]",
-            self.tag,
-            request,
-            ctx.exec_ctx.as_ref().unwrap().term,
-            ctx.exec_ctx.as_ref().unwrap().index
-        );
+        if cmd_type != AdminCmdType::CompactLog && cmd_type != AdminCmdType::CommitMerge {
```

Comment:
why ignore CommitMerge?

Reply:
It's a special logic for release 3.0. It's removed for release 2.1 now.